internal-api/wp-cli-utils-http-request.md

```diff
@@ -31,6 +32,7 @@ if ( 20 != substr( $md5_response->status_code, 0, 2 ) ) {
      WP_CLI::error( "Couldn't access md5 hash for release (HTTP code {$response->status_code})" );
 }
 ```
+                           or in the body for POST requests.
```

Comment:
@schlessera Where is this coming from? 🤔  Looks a bit out of place

Reply:
Hmm, good catch. Looks like the parsing fails here.

![image](https://github.com/wp-cli/handbook/assets/83631/69e51f07-d3ec-48ed-a46b-136f8708cea5)